shared/common-adapters/floating-menu/menu-layout/index.native.js

```diff
@@ -90,7 +90,6 @@ const styles = Styles.styleSheetCreate({
     backgroundColor: Styles.globalColors.white,
     justifyContent: 'flex-end',
     paddingBottom: Styles.globalMargins.tiny,
-    paddingTop: Styles.globalMargins.small,
```

Comment:
cc @chrisnojima I reverted [this change](https://github.com/keybase/client/pull/15415/files#diff-f9f01aea8b8cd5f7f0a38f28126fded6R93) because of this:
<img width="446" alt="image" src="https://user-images.githubusercontent.com/11968340/51079028-f8b01580-168d-11e9-8cdd-7d5efe5e4405.png">